apps/meteor/ee/app/apps/bridges/livechat.ts

```diff
@@ -24,7 +25,7 @@ export class AppLivechatBridge extends LivechatBridge {
 	}
 
 	protected isOnline(departmentId?: string): boolean {
-		return Livechat.online(departmentId);
+		return Future.fromPromise(Livechat.online(departmentId)).wait() as boolean;
```

Comment:
Promise.await was not enough? :oh-no:

Reply:
`Promise.await` won't be available in the Microservice scope, so we need to go deeper